components/brave_rewards/resources/ui/components/checkout/dialogFrame/style.ts

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled, { css } from 'styled-components'
+
+import backgroundSrc from './assets/dialog_bg.svg'
+
+export const MainPanel = styled.div<{ showBackground?: boolean }>`
+  overflow: hidden;
+  position: relative;
+  top: 0;
+  left: 0;
+  max-width: 548px;
+  background: ${p => p.theme.color.panelBackground};
+  font-weight: normal;
+  font-family: ${p => p.theme.fontFamily.heading};
+  font-size: 14px;
+  color: ${p => p.theme.palette.black};
+
+  ${p => !p.showBackground ? '' : css`
+    background-image: url(${backgroundSrc});
+    background-repeat: no-repeat;
+    background-position: bottom right;
+  `}
+
+  a {
```

Comment:
this can lead in overrides, do we need it here?

Reply:
I think it's easier to set a default link style here than to create inline styles for every link.

Reply:
ok